ios/Exponent/Kernel/Views/Loading/EXManagedAppSplashScreenController.h

```diff
@@ -0,0 +1,18 @@
+//
+//  EXManagedAppSplashScreenController.h
+//  Expo Go (unversioned)
+//
+//  Created by andrew on 2021-06-07.
+//  Copyright © 2021 650 Industries. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <EXSplashScreen/EXSplashScreenViewController.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface EXManagedAppSplashScreenController : EXSplashScreenViewController
```

Comment:
As we're having `EXSplashScreenViewController`, let's also have:
```suggestion
@interface EXManagedAppSplashScreenViewController : EXSplashScreenViewController
```